docs/multi-echo.rst

```diff
@@ -462,6 +462,19 @@ time point.
     research to better understand how these steps can be applied before denoising.
 
 
+4. Apply susceptibility distortion correction consistently across echoes
+========================================================================
+
+One key feature of susceptibility distortion is that it is primarily a factor of echo spacing, rather than echo time.
```

Comment:
```suggestion
One key feature of susceptibility distortion is that it is primarily a factor of readout pattern and total readout time, rather than echo time.
```